RecoCTPPS/TotemRPLocal/src/TotemTimingRecHitProducerAlgorithm.cc

```diff
@@ -191,5 +191,5 @@ float TotemTimingRecHitProducerAlgorithm::constantFractionDiscriminator(const st
                        (dataProcessed[indexOfThresholdCrossing - 1] - dataProcessed[indexOfThresholdCrossing]) *
                        (threshold - dataProcessed[indexOfThresholdCrossing]) +
                    time[indexOfThresholdCrossing]
-             : TotemTimingRecHit::NO_T_AVAILABLE;
+             : (float)TotemTimingRecHit::NO_T_AVAILABLE;
```

Comment:
+1

Reply:
maybe in the actual comment section :)

Reply:
@mrodozov I don’t have any particular right for the review process, just to say this cast is actually what is required on this side of the PPS code :-)